ArcGISRuntimeSDKQt_QMLSamples/Layers/ExportVectorTiles/ExportVectorTiles.qml

```diff
@@ -274,6 +284,28 @@ Rectangle {
             // Start the export job once export parameters have been created
             exportVectorTilesJob.start();
         }
+
+        readonly property Timer timer: Timer {
+            id: jobCancelDoneTimer
+            interval: 2000
+            onTriggered: { exportProgressWindow.visible = false; statusText.visible = true }
+        }
+
+        Connections {
+            id: exportVectorTilesJobConnections
+            ignoreUnknownSignals: true
+            function onCancelAsyncTaskStatusChanged() {
+                if(exportVectorTilesTask.exportVectorTilesJob.cancelAsyncTaskStatus === Enums.TaskStatusCompleted)
+                    statusTextCanceled.text = "Job canceled successfully"
+                else if(exportVectorTilesTask.exportVectorTilesJob.cancelAsyncTaskStatus === Enums.TaskStatusCompleted)
+                    statusTextCanceled.text = "Job canceled not successfully"
```

Comment:
I would update the error message to something like: "Job failed to cancel successfully"

Reply:
I have noticed the not spinning busy indicator and put a fix in, surprised to see it not working again!